### ISSUE TITLE
Customizable Intervals for Usage Breaker Reminders

### ISSUE THREAD

**Issue by chiemii** [8bitPit/Niagara-Issues#3094]
### Is your feature request related to a problem?

I often find myself reopening an app right after closing it, even though I wanted to take a break. The current Usage Breaker timers are great for preventing long app sessions, but they don’t help with stopping those quick, impulsive reopens. It’d be awesome to have something that addresses this issue.

### Describe the solution you'd like

Customizable reminder intervals. Instead of being limited to the preset times (5, 10, 20 minutes, etc.), an option that would let us pick intervals that would fit better for someone more easily distracted. Or even just a toggle-able option that really just gets up on your face if you get on the leisure apps if you were planning on doing deep work!

### Feature Request Procedure

- [X] I have read the [knowledge base article on feature requests](https://help.niagaralauncher.app/article/44-feature-requests) and followed the steps mentioned

**Comment by amerikiwi93** [comment]
Yes, it would be useful if the usage breaker notification could be permanently active if you have spend over 5 mins on the unproductive apps in the current session or 30 mins for the whole day.

**Comment by ssssamstrup** [comment]
For a short-term solution use an app-blocker (I use Stay Focused) to set a limit on those apps for 0min/day. Then when you try to open it, your attempt will get blocked right away. To open the app again, you'll then need to open the app-blocker and allow yourself what more time you want (5, 10, 15, 30 mins are options in Stay Focused) and can open the app again. When the time has gone, the app will be blocked again and you can't open it straight away.

I recommend using the 0min/day as a hard limit to break your habit of opening the app again imidiately. For apps that you often open by habit this works even better to break that habit. 
For apps that don't respect your time and attention (see all ad-based scrolling-feed platforms (Fb, Instagram, News, Tik-tok) this has been the only way me and my friend-group have found to break these habits effectively without outright uninstalling the apps.

Hope my unsoliticed advice is helpful. 😆
